src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    public final int STACK_SIZE = 12;
+    public final int RETURN = -1;
+    public final int IS_FULL = 11;
```

Comment:
This constant is related to STACK_SIZE. Changing only one leads to an error (inconsistency).